documentation/modules/exploit/unix/webapp/joomla_sqli_rce_3_7_0.md

```diff
@@ -0,0 +1,49 @@
+## Vulnerable Application
+
+  This module exploits a sql injection in the core of Joomla 3.7.0.
+  This vulnerability can allow remote code execution.
```

Comment:
With the SQLi, its possible to enumerate cookies of administrative users, and hijack one of their sessions.  If no administrators are authenticated, the RCE portion will not work.  If a session hijack is available, one of the website templates is identified, and our payload is added to the template as a new file, and then executed.

Reply:
admin or super user?